user_input.rb

```diff
@@ -0,0 +1,22 @@
+# rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
- [ ] @Team It's not always good practice to disable the linter check in our files, linters help us write quality code, kindly consider fixing the linter error instead of disabling the linter check.